src/settings/helpers/FeatureSettingWrapper.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import SettingsStore from '../SettingsStore';
+import { UIFeature } from '../UIFeature';
+
+interface Props {
+    feature: UIFeature;
```

Comment:
there's a bit of a naming conflict here: can we call it a `uiFeature` or similar? "Feature" is typically associated with a labs flag for historical context.